templates/microk8s/compare.html

```diff
@@ -0,0 +1,304 @@
+{% extends 'base_index.html' %}
+
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
+{% block title %}MicroK8s vs k3s vs Minikube{% endblock %}
+
+{% block meta_description %}
+  Feature comparison of lightweight Kubernetes distributions: MicroK8s, K3s and minikube.
+{% endblock meta_description %}
+
+{% block meta_image %}
+  https://assets.ubuntu.com/v1/920467d2-Compare_MicroK8s-K3S-Minikube.svg
+{% endblock meta_image %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1wWos3iWMTCqDbxZBDuSVqvY6ZwYux-goCrX0L0E5Weo/edit
+{% endblock %}
+
+{% block content %}
+
+  {% call(slot) vf_hero(
+    title_text='MicroK8s<br class="u-hide--small" /> vs K3s vs minikube',
+    layout='50/50'
+    ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Lightweight Kubernetes distributions are becoming increasingly popular for local development, edge/IoT container management and self-contained application deployments. If you are looking to run Kubernetes on devices lighter in resources, have a look at the table below. MicroK8s stands out for its simplicity, robustness and completeness.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="/microk8s/contact-us"
+         class="p-button--positive js-invoke-modal"
+         aria-controls="microk8s-modal">Get in touch</a>
+    {%- endif -%}
+  {% endcall -%}
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-section--shallow">
+        <h2>Compare features</h2>
+      </div>
+      <div class="table-container">
+        <table>
+          <thead>
+            <tr>
+              <th style="width: 30%;">Feature</th>
+              <th>
+                {{ image(url="https://assets.ubuntu.com/v1/78679228-microk8s.svg",
+                                alt="MicroK8s",
+                                width="189",
+                                height="45",
+                                hi_def=True,
+                                loading="auto") | safe
+                }}
+              </th>
+              <th>
+                {{ image(url="https://assets.ubuntu.com/v1/d07473a1-k3s.svg",
+                                alt="K3s",
+                                width="189",
+                                height="45",
+                                hi_def=True,
+                                loading="auto") | safe
+                }}
+              </th>
+              <th>
+                {{ image(url="https://assets.ubuntu.com/v1/268abf4b-minikube-on-grey.png",
```

Comment:
This image has a grey background:

<img width="421" height="147" alt="Image" src="https://github.com/user-attachments/assets/91fda205-d140-4672-8441-a52b13eae547" />

Perhaps the page body needs the `is-paper` class